zulipterminal/ui_tools/views.py

```diff
@@ -1178,7 +1189,23 @@ def _fetch_user_data(controller: Any, user_id: int) -> Dict[str, Any]:
             if data["last_active"]:
                 display_data["Last active"] = data["last_active"]
 
-        return display_data
+        custom_profile_data = data["custom_profile_data"]
```

Comment:
At the risk of increased nesting we could nest all of this section in the non-bot area, since bots cannot have custom fields, but a comment should suffice since the data should simply not have the fields in the first place.